CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2025.6.2 (25 June 2025) Release
```

Comment:
@debonte is this the right order? By date?

Reply:
I think in the past we've sorted by version number because that's what people are generally searching for.